WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -146,9 +154,12 @@ final class DashboardViewModel: ObservableObject {
                 await self?.syncDashboardEssentialData()
             }
             group.addTask { [weak self] in
-                guard let self = self else { return }
+                guard let self else { return }
                 await reloadCards(showOnDashboardCards)
             }
+            group.addTask { [weak self] in
+                await self?.checkInboxEligibility()
+            }
```

Comment:
Would it be better to move it under `syncDashboardEssentialData` method? 🤔

Reply:
Similar comment as above about the complexity when triggering `syncDashboardEssentialData` in unit tests. I'm keeping this implementation for simplicity now.